src/t8_forest/t8_forest_vtk.h

```diff
@@ -32,9 +32,21 @@
 #include <t8_forest/t8_forest_general.h>
 #include <t8_forest/t8_forest_geometrical.h>
 
+#if T8_WITH_VTK
+#include <vtkUnstructuredGrid.h>
+#endif
+
 T8_EXTERN_C_BEGIN ();
 /* function declarations */
 
+#if T8_WITH_VTK
+void
+t8_forest_to_vtkUnstructuredGrid (t8_forest_t forest, vtkSmartPointer<vtkUnstructuredGrid> unstructuredGrid,
```

Comment:
Is this function part of the public/internal API? Please consider to add documentation to this function declaration.

Reply:
A short sidenote: This isn't introduced by this PR, the function existed befor: Nevertheless I wrote it and didn't document it, so I will do so ;)